common/src/main/java/org/apache/celeborn/common/network/server/BufferStreamManager.java

```diff
@@ -426,21 +478,19 @@ public boolean releaseStream(Long streamId) {
     }
 
     public void close() {
-      logger.info("release map data partition {}", fileInfo);
+      logger.debug("release map data partition {}", fileInfo);
 
       IOUtils.closeQuietly(dataFileChanel);
       IOUtils.closeQuietly(indexChannel);
-
-      if (this.buffers != null) {
-        for (ByteBuf buffer : this.buffers) {
-          memoryManager.recycleReadBuffer(buffer);
-        }
-      }
-
-      this.buffers = null;
+      bufferQueue.release();
 
       isReleased = true;
     }
+
+    @Override
+    public String toString() {
+      return "MapDataPartition{" + "fileInfo=" + fileInfo.getFilePath() + '}';
+    }
   }
 
   class StorageFetcherPool {
```

Comment:
I think we don't need the class. Just put executorPools and getExecutorPool() to BufferStreamManager